### PR TITLE
Add support for specifying ssh user

### PR DESCRIPTION
**Pull request by satterly**
**Example**

```
$ ssm ssh -i i-066ea79153a164de8 --profile ophan --private --user hadoop
```

### CODE REVIEW

**Comment by shtukas** [comment]
@satterly 
Do you mind if I replace `-U, --user <user>` with `-u, --user <user>` ?

**Comment by shtukas** [comment]
Awesome. Thanks!

File: src/main/scala/com/gu/ssm/models.scala [hunk 1]
```diff
@@ -17,18 +17,22 @@ case class Arguments(
...
   region: Region,
   mode: Option[SsmMode],
+  user: Option[String],
   singleInstanceSelectionMode: SingleInstanceSelectionMode,
   isSelectionModeNewest: Boolean,
   isSelectionModeOldest: Boolean,
   usePrivateIpAddress: Boolean
 )
 object Arguments {
+  val DefaultUser = "ubuntu"
```
Comment (adamnfish) [inline src/main/scala/com/gu/ssm/models.scala]: Should be lower case for a val `defaultUser`

Comment (shtukas) [reply src/main/scala/com/gu/ssm/models.scala]: True!

Comment (shtukas) [reply src/main/scala/com/gu/ssm/models.scala]: https://github.com/guardian/ssm-scala/pull/45

**Pull request merged**